### PR TITLE
Removing KUDO references and Warning on Deprecated use of kudo.dev

### PR DESCRIPTION
**Pull request by kensipe**
Signed-off-by: Ken Sipe <kensipe@gmail.com>

### CODE REVIEW

**Review by ANeumann82 — APPROVED** [review]
🚢

**Review by nfnt — APPROVED** [review]
LGTM!

File: pkg/test/utils/kubernetes.go [hunk 1]
```diff
@@ -392,6 +392,9 @@ func ConvertUnstructured(in runtime.Object) (runtime.Object, error) {
 	// Deprecated: kudo.dev is deprecated.
 	kudoGroup := "kudo.dev"
 	kuttlGroup := "kuttl.dev"
+	if group == kudoGroup {
```
Comment (alenkacz) [inline pkg/test/utils/kubernetes.go:395]: with this change, this warning is printed for all the resources from KUDO (try running KUDO tests now and you'll see). I am not sure this was the right place to put this warning to.

Comment (nfnt) [reply pkg/test/utils/kubernetes.go:395]: 👍 
This function is also called when applying general resources. These could be KUDO CRDs or CRs where `kudo.dev` is expected.

Comment (alenkacz) [reply pkg/test/utils/kubernetes.go:395]: yeah this is what kudo tests say right now
```
2020/10/26 13:56:54 WARNING: kudo.dev group has been deprecated and is schedule to be removed in KUTTL 0.8.0
2020/10/26 13:56:54 WARNING: kudo.dev group has been deprecated and is schedule to be removed in KUTTL 0.8.0
```

**Pull request merged**